astropy/utils/data.py

```diff
@@ -1159,15 +1159,11 @@ def download_files_in_parallel(urls, cache=False, show_progress=True,
     else:
         progress = io.BytesIO()
 
-    if timeout is None:
```

Comment:
I think that was just "dead code", the `timeout is None` case was already covered a few lines above.